kapitan/targets.py

```diff
@@ -97,10 +103,14 @@ def compile_targets(
                     inventory_path, target_objs, dep_cache_dir, kwargs.get("force_fetch", False), pool
                 )
                 cached.reset_inv()
-                target_objs = load_target_inventory(inventory_path, updated_targets)
+                target_objs = load_target_inventory(
+                    inventory_path, updated_targets, ignore_class_notfound=True
+                )
                 cached.inv_sources.update(new_sources)
                 new_sources = list(set(list_sources(target_objs)) - cached.inv_sources)
-
+            # reset inventory cache and load target objs to check for missin classes
+            cached.reset_inv()
+            target_objs = load_target_inventory(inventory_path, updated_targets, ignore_class_notfound=True)
```

Comment:
I believer this line should be `ignore_class_notfound=False` ?
also typo in comment above `missin classes`

Reply:
Yes, that should be False. A few more issues have come up during testing and I'm trying to resolve it asap. Will make push the code and ping you as soon as it is ready for review :)